hub.py

```diff
@@ -149,7 +149,20 @@ def get_generated_config(self, auth_provider, proxy_secret_key):
                         'name': self.cluster.spec['image_repo']
                     },
                 },
-            }
+                'hub': {
+                    'extraContainers': {
+                        'templates-sync': {
+                            'args':
+                                '-c',
+                                f'while true; do git fetch origin; \
```

Comment:
Consider putting this out into its own triple quoted string (possibly with a [dedent](https://docs.python.org/3/library/textwrap.html#textwrap.dedent)) instead of it being split out here.

We've had to substitute the hub name in a few places - here, nfs share name, etc. We should open an issue about being able to do that declaratively, so we could've left this in values.yaml.